src/engraving/draw/qpainterprovider.cpp

```diff
@@ -186,35 +186,41 @@ void QPainterProvider::drawPath(const QPainterPath& path)
     m_painter->drawPath(path);
 }
 
-void QPainterProvider::drawPolygon(const QPointF* points, int pointCount, PolygonMode mode)
+void QPainterProvider::drawPolygon(const PointF* points, int pointCount, PolygonMode mode)
 {
+    IF_ASSERT_FAILED(sizeof(QPointF) == sizeof(PointF)) {
```

Comment:
Isn't this a situation where you could also use `static_assert`?

Reply:
Good point! Changed to use static_assert